lib/schemas/nsm/v1.json

```diff
@@ -1138,6 +1138,13 @@
       "x-pii": false,
       "type":"string"
     },
+    "import_date": {
+      "examples": [
+        "2024-10-04T08:51:26.791Z"
+      ],
+      "x-pii": false, 
+      "type": "string"
```

Comment:
This can be nullable so you have to specify `null` in the type list eg      ` "type":["null","string"]`